scene/gui/split_container.cpp

```diff
@@ -410,53 +402,47 @@ void SplitContainer::_resort_children() {
 	dragger_control->queue_redraw();
 }
 
-void SplitContainer::_compute_middle_sep(Control *p_first, Control *p_second, int p_sep) {
-	bool first_visible = p_first && p_first->is_visible();
-	bool second_visible = p_second && p_second->is_visible();
+void SplitContainer::_compute_middle_sep(bool p_clamp) {
```

Comment:
I think with this you are calling _get_child(0) and second here, but the previous way it was using the cached node.

Reply:
It will be called from inside the SplitContainer dragger control child and we will do the same but instead we will need to get the parent children, maybe we can store them in a variable later inside the class instead of using _git_child() more than once each frame to make it run  faster and we also can unify the code, i can go through the whole code and make a lot of cleanup later but for this PR i just need to make sure that it won't introduce any other issues and it's issue free.